src/test/java/org/trellisldp/event/EventSerializerTest.java

```diff
@@ -64,6 +65,7 @@ public void setUp() {
         when(mockEvent.getTypes()).thenReturn(singleton(Create));
         when(mockEvent.getTargetTypes()).thenReturn(singleton(Container));
         when(mockEvent.getInbox()).thenReturn(of(rdf.createIRI("info:ldn/inbox")));
+        when(mockEvent.getCreated()).thenReturn(now());
```

Comment:
What I might do here is create a class-level field:

    private final Instant time = now();

And then return that value in your mock:

    when(mockEvent.getCreated()).thenReturn(time);

That way you can do a value test in the JUnit tests below.